docs/release-notes/NuGet-6.2.md

```diff
@@ -0,0 +1,96 @@
+|:---|:---|:---|
+| [**6.2.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.2](https://visualstudio.microsoft.com/downloads/) | [6.0.202](https://dotnet.microsoft.com/download/dotnet-core/6.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.2
+
+* Add TFM for .NET nanoFramework - [#10800](https://github.com/NuGet/Home/issues/10800)
+
+* [Feature]: Require package source mapping when using CPM - [#11505](https://github.com/NuGet/Home/issues/11505)
+
+* [Feature]: Allow overriding a centrally defined package version - [#11516](https://github.com/NuGet/Home/issues/11516)
+
+* [Feature]: Add support for a dedicated environment variable providing the NuGetScratch path. - [#11671](https://github.com/NuGet/Home/issues/11671)
+
+* [Feature]: Add IVsNuGetProjectUpdateEvents in Visual Studio, reporting of restore changes for PackageReference based projects.  - [#9782](https://github.com/NuGet/Home/issues/9782)  - [See documentation](https://docs.microsoft.com/nuget/visual-studio-extensibility/nuget-api-in-visual-studio#ivsnugetprojectupdateevents-interface)
+
+### Issues fixed in this release
+
+**DCRs:**
+
+* Make LocalPackageFileCache methods virtual - [#10325](https://github.com/NuGet/Home/issues/10325)
+
+* NuGetScratch lock files are not cleaned up - [#10679](https://github.com/NuGet/Home/issues/10679)
+
+* AutoCompleteResourceV3 does not use the supplied logger - [#11272](https://github.com/NuGet/Home/issues/11272)
+
+* [DCR]: Mitigate missing nuget.org when non-NuGet tool creates nuget.config without any sources - [#11387](https://github.com/NuGet/Home/issues/11387)
+
+* Add Author to the tooltip for a package in the packages list of PM UI - [#11499](https://github.com/NuGet/Home/issues/11499)
+
+* Remove NU5049  - [#11598](https://github.com/NuGet/Home/issues/11598)
+
+**Bugs:**
+
+* Add support for grouping to the InfiniteScrollList, allowing it to be enabled or disabled - [#10748](https://github.com/NuGet/Home/issues/10748)
+
+* Make the InfiniteScrollList grouping sections expandable and collapsible - [#10749](https://github.com/NuGet/Home/issues/10749)
+
+* Read and store the transitive origins of a package while reading installed packages from assets file - [#10751](https://github.com/NuGet/Home/issues/10751)
+
+* Add caching of the transitive dependencies data pulled from the lockfile (assets file) - [#10752](https://github.com/NuGet/Home/issues/10752)
+
+* Surface the transitive packages and its transitive origins through the search layer - [#11486](https://github.com/NuGet/Home/issues/11486)
+
+* NuGet.exe list from local packages folder does not work with the AllVersion flag - [#4537](https://github.com/NuGet/Home/issues/4537)
+
+* Project A referencing package B via AssetTargetFallback, doesn't use that same AssetTargetFallback to pull B's dependency package C - [#5957](https://github.com/NuGet/Home/issues/5957)
```

Comment:
I wrote up a guide for the ATF change, maybe we can link it? https://github.com/NuGet/Samples/pull/53

In particular I want to: 

* Get the samples PR reviewed and merged.
* Add a comment linking that here. 


cc @JonDouglas 

Do you think we should add this to the notable changes list? 

Do we want to add a `Known issues` section or something like that? 

Maybe we have a better that mentions that this issue has consequences. 

Open to ideas.

Reply:
The link should be https://github.com/NuGet/Samples/tree/main/AssetTargetFallbackTransitiveDependencies.

Reply:
@dominoFire suggests that we can add `More info` and link to the example, in particular https://github.com/NuGet/Samples/tree/main/AssetTargetFallbackTransitiveDependencies.

Reply:
Pushed this. 

Should be complete now.

Reply:
`known issues` best for anything that is known with the release and will addressed soon or has significant limitations when using today.

Reply:
This one is basically, you *might* have some weirdness, but there's no proposed changes right now.

See More Information, what we have right now, might work better for that. 

Feel free to tweak if you think it's necessary